Sources/TuistGenerator/Generator/WorkspaceGenerator.swift

```diff
@@ -88,10 +88,22 @@ final class WorkspaceGenerator: WorkspaceGenerating {
         /// Projects
 
         var generatedProjects = [AbsolutePath: GeneratedProject]()
-        try graph.projects.forEach { project in
+        
+        func filterCarthageProjectIfRequired(project: AbsolutePath) -> Bool {
```

Comment:
Should CarthageNode be bubbled up here to filter on the node type instead of the path containing `"Carthage/Checkouts"`